src/plugins/data/server/index_patterns/fetcher/lib/field_capabilities/field_caps_response.test.js

```diff
@@ -24,7 +24,7 @@ import sinon from 'sinon';
 import * as shouldReadFieldFromDocValuesNS from './should_read_field_from_doc_values';
 import { shouldReadFieldFromDocValues } from './should_read_field_from_doc_values';
 
-import { getKbnFieldType } from '../../../../../../plugins/data/common';
+import { getKbnFieldType } from '../../../../../common';
```

Comment:
I would replace the import path to  '../../../../../../data/common' for clarity.

Reply:
Done.